lib/elixir/lib/calendar.ex

```diff
@@ -0,0 +1,87 @@
+defmodule Calendar do
```

Comment:
>     We can rename the Calendar module in Elixir to something else.
>
>     We can rename the Calendar module in calendar to something else, like Calendar.Types.
> 

The Calendar module is only used for one macro. The macro is used to alias e.g. Calendar.Date and Calendar.DateTime. This is done mostly in order to call functions like `Calendar.Date.diff` as just `Date.diff`.

If Elixir 1.3 defines `Date` (without namespacing), then it makes no sense to alias `Calendar.Date`. Thus there is no point in having a macro for aliasing them in the Calendar module nor in a renamed module.

I guess people will just have to use the entire module name, e.g. `Calendar.Date.diff`.

>     We can add the __using__ function currently defined in your Calendar module to this module with a deprecation warning. This means that, when users migrate to Elixir 1.3, if they use Calendar, they will get a proper deprecation warning. We would remove the deprecation by Elixir 1.4.
> 

This is a good idea.